docs/buildah-build.1.md

```diff
@@ -411,6 +411,24 @@ By default, Buildah manages _/etc/hosts_, adding the container's own IP address.
 
 Set the OS of the image to be built, and that of the base image to be pulled, if the build uses one, instead of using the current operating system of the host.
 
+**--output**, **-o**=""
+
+Output destination (format: type=local,dest=path)
+
+By default, a local container image is created from the build result. The --output (or -o) option allows you to override this behavior and specify a custom exporter. For example, custom exporters allow you to export the build artifacts as files on the local filesystem instead of a container image, which can be useful for generating local binaries, code generation, etc.
```

Comment:
This suggests that when this option is used, we don't commit a container image, but it looks like we still do.

Reply:
Fixed explicitly mentioned.

Reply:
This still reads to me as though using this option causes the builder to not commit an image, as committing the image is the default behavior that's being overridden when this option is specified.

Reply:
fixed it after confirming from @nalind hence marking resolved again.